File: Day 26 : Nested Logic/Solution.py
if __name__ == "__main__":
    d1, m1, y1 = input().split()
    d2, m2, y2 = input().split()
    d1, m1, y1 = int(d1), int(m1), int(y1)
    d2, m2, y2 = int(d2), int(m2), int(y2)
    
    if(y1 > y2):
        print(10000)
    elif (y1 < y2):
        print(0)
    elif(y1 == y2):
        if(m1 <= m2):
            if(d1 <= d2):
                print(0)
            else:
                print(abs(15 * (d1 - d2)))
        else:
            print(abs(500 * (m1 - m2)))

File: Day 26 : Nested Logic/Solution.js
function processData(input) {
        var date = input.split(" ");
        let retDay = parseInt(date[0]);
        let retMon = parseInt(date[1]);
        let temp = date[2].split("\n");
        let retYr = parseInt(temp[0]);
        let dueDay = parseInt(temp[1]);
        let dueMon = parseInt(date[3]);
        let dueYr = parseInt(date[4]);

        let fine = 0;

        if (dueYr == retYr) {
                if (dueMon == retMon) {
                        if (retDay > dueDay) {
                                fine = 15 * (retDay - dueDay);
                        }
                } else if (retMon > dueMon) {
                        fine = 500 * (retMon - dueMon);
                }
        } else if (retYr > dueYr) {
                fine = 10000;
        }
        console.log(fine);
}
